tutorials/xr/xr_action_map.rst

```diff
@@ -0,0 +1,242 @@
+
+The concept here is that you could have different sets that provide bindings in different scenarios. You could have a set ``Character control`` for when you're walking around, ``Vehicle control`` for when you're operating a vehicle, ``Menu`` for when a menu is open, etc.
+Only the action set applicable to the current state of your game/application can then be enabled.
+
+This is especially important if you wish to bind the same input on a controller to a different action. For instance:
+
+  * in your ``Character control`` set you may have an action ``Jump``, 
+  * in your ``Vehicle control`` set you may have an action ``Accelerate``, 
+  * in your ``Menu`` set you may have an action ``Select``.
+
+All are bound to the trigger on your controller.
+
+OpenXR will only bind an input or output to a single action. If the same input or output is bound to multiple actions the one in the active action set with the highest priority will be the one updated/used. So in our above example it will thus be important that only one action set is active.
+
+For your first XR game/application we highly recommend starting with just a single action set and not over-engineer things.
+
+For our walk through in this document we will thus create a single action set called ``my_first_action_set``, we do this by pressing the ``Add action set`` button:
+
+.. image:: img/xr_my_first_action_set.webp
+
+The columns in our table are as follows:
+
+.. list-table::
+  :class: wrap-normal
+  :width: 100%
+  :widths: 7 23 70
+  :header-rows: 1
+
+  * - Col
+    - Value
+    - Description
+  * - 1
+    - my_first_action_set
+    - This is the internal name of the action set. OpenXR doesn't specify specific restrictions on this name other then size, however some XR runtimes will not like spaces or special characters.
+  * - 2
+    - My first action set
+    - This is a human-readable name for the action set. Some XR runtimes will display this name to the end user, for example in configuration dialogs.
+  * - 3
+    - 0
+    - This is the priority of the action set. If multiple active action sets have actions bound to the same controllers inputs or outputs, the action set with the highest priority value will determine the action that is updated.
+
+Actions
+=======
+
+In the XR action map, actions are the entities that your game/application will interact with. For instance, we can define an action ``Shoot`` and the input bound to that action will trigger the ``button_pressed`` signal on the relevant :ref:`XRController3D <class_xrcontroller3d>` node in your scene with ``Shoot`` as the ``name`` parameter of the signal.
+
+You can also poll the current state of an action. :ref:`XRController3D <class_xrcontroller3d>` for instance has an ``is_button_pressed`` method.
+
+Actions can be used for both in- and output. Each action has a type that defines the type of in- or output it can operate on.
```

Comment:
Should be better to fully write `input` instead of using `in-`

Reply:
yes agree, much clearer